# Subset: Sequencing

URI: [Sequencing](Sequencing)





## Identifier and Mapping Information







### Schema Source


* from schema: https://turbomam.github.io/mixs-envo-struct-knowl-extraction//GSC_MIxS_6




## Classes in subset

| Class | Description |
| --- | --- |




## Slots in subset

| Slot | Description |
| --- | --- |
| [adapters](adapters.md) | Adapters provide priming sequences for both amplification and sequencing of t... |
| [annot](annot.md) | Tool used for annotation, or for cases where annotation was provided by a com... |
| [assembly_name](assembly_name.md) | Name/version of the assembly provided by the submitter that is used in the ge... |
| [assembly_qual](assembly_qual.md) | The assembly quality category is based on sets of criteria outlined for each ... |
| [assembly_software](assembly_software.md) | Tool(s) used for assembly, including version number and parameters |
| [associated_resource](associated_resource.md) | A related resource that is referenced, cited, or otherwise associated to the ... |
| [bin_param](bin_param.md) | The parameters that have been applied during the extraction of genomes from m... |
| [bin_software](bin_software.md) | Tool(s) used for the extraction of genomes from metagenomic datasets, where p... |
| [chimera_check](chimera_check.md) | Tool(s) used for chimera checking, including version number and parameters, t... |
| [compl_appr](compl_appr.md) | The approach used to determine the completeness of a given genomic assembly, ... |
| [compl_score](compl_score.md) | Completeness score is typically based on either the fraction of markers found... |
| [compl_software](compl_software.md) | Tools used for completion estimate, i |
| [contam_score](contam_score.md) | The contamination score is based on the fraction of single-copy genes that ar... |
| [contam_screen_input](contam_screen_input.md) | The type of sequence data used as input |
| [contam_screen_param](contam_screen_param.md) | Specific parameters used in the decontamination sofware, such as reference da... |
| [decontam_software](decontam_software.md) | Tool(s) used in contamination screening |
| [detec_type](detec_type.md) | Type of UViG detection |
| [feat_pred](feat_pred.md) | Method used to predict UViGs features such as ORFs, integration site, etc |
| [host_pred_appr](host_pred_appr.md) | Tool or approach used for host prediction |
| [host_pred_est_acc](host_pred_est_acc.md) | For each tool or approach used for host prediction, estimated false discovery... |
| [lib_layout](lib_layout.md) | Specify whether to expect single, paired, or other configuration of reads |
| [lib_reads_seqd](lib_reads_seqd.md) | Total number of clones sequenced from the library |
| [lib_screen](lib_screen.md) | Specific enrichment or screening methods applied before and/or after creating... |
| [lib_size](lib_size.md) | Total number of clones in the library prepared for the project |
| [lib_vector](lib_vector.md) | Cloning vector type(s) used in construction of libraries |
| [mag_cov_software](mag_cov_software.md) | Tool(s) used to determine the genome coverage if coverage is used as a binnin... |
| [mid](mid.md) | Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to spe... |
| [nucl_acid_amp](nucl_acid_amp.md) | A link to a literature reference, electronic resource or a standard operating... |
| [nucl_acid_ext](nucl_acid_ext.md) | A link to a literature reference, electronic resource or a standard operating... |
| [number_contig](number_contig.md) | Total number of contigs in the cleaned/submitted assembly that makes up a giv... |
| [otu_class_appr](otu_class_appr.md) | Cutoffs and approach used when clustering species-level OTUs |
| [otu_db](otu_db.md) | Reference database (i |
| [otu_seq_comp_appr](otu_seq_comp_appr.md) | Tool and thresholds used to compare sequences when computing "species-level" ... |
| [pcr_cond](pcr_cond.md) | Description of reaction conditions and components of PCR in the form of 'init... |
| [pcr_primers](pcr_primers.md) | PCR primers that were used to amplify the sequence of the targeted gene, locu... |
| [pred_genome_struc](pred_genome_struc.md) | Expected structure of the viral genome |
| [pred_genome_type](pred_genome_type.md) | Type of genome predicted for the UViG |
| [reassembly_bin](reassembly_bin.md) | Has an assembly been performed on a genome bin extracted from a metagenomic a... |
| [ref_db](ref_db.md) | List of database(s) used for ORF annotation, along with version number and re... |
| [sc_lysis_approach](sc_lysis_approach.md) | Method used to free DNA from interior of the cell(s) or particle(s) |
| [sc_lysis_method](sc_lysis_method.md) | Name of the kit or standard protocol used for cell(s) or particle(s) lysis |
| [seq_meth](seq_meth.md) | Sequencing machine used |
| [seq_quality_check](seq_quality_check.md) | Indicate if the sequence has been called by automatic systems (none) or under... |
| [sim_search_meth](sim_search_meth.md) | Tool used to compare ORFs with database, along with version and cutoffs used |
| [sop](sop.md) | Standard operating procedures used in assembly and/or annotation of genomes, ... |
| [sort_tech](sort_tech.md) | Method used to sort/isolate cells or particles of interest |
| [target_gene](target_gene.md) | Targeted gene or locus name for marker gene studies |
| [target_subfragment](target_subfragment.md) | Name of subfragment of a gene or locus |
| [tax_class](tax_class.md) | Method used for taxonomic classification, along with reference database used,... |
| [tax_ident](tax_ident.md) | The phylogenetic marker(s) used to assign an organism name to the SAG or MAG |
| [trna_ext_software](trna_ext_software.md) | Tools used for tRNA identification |
| [trnas](trnas.md) | The total number of tRNAs identified from the SAG or MAG |
| [vir_ident_software](vir_ident_software.md) | Tool(s) used for the identification of UViG as a viral genome, software or pr... |
| [wga_amp_appr](wga_amp_appr.md) | Method used to amplify genomic DNA in preparation for sequencing |
| [wga_amp_kit](wga_amp_kit.md) | Kit used to amplify genomic DNA in preparation for sequencing |
| [x16s_recover](x16s_recover.md) | Can a 16S gene be recovered from the submitted SAG or MAG? |
| [x16s_recover_software](x16s_recover_software.md) | Tools used for 16S rRNA gene extraction |


## Enumerations in subset

| Enumeration | Description |
| --- | --- |

